.circleci/config.yml

```diff
@@ -187,17 +187,19 @@ jobs:
           no_output_timeout: 60m
           command: |
             pids=()
+
             npx nx-cloud record -- nx format:check --base=$NX_BASE --head=$NX_HEAD &
             pids+=($!)
 
-            npx nx run-many -t check-imports check-commit check-lock-files root-lint depcheck documentation --parallel=1 --no-dte &
+            npx nx run-many -t check-imports check-commit check-lock-files lint depcheck documentation --parallel=1 --no-dte &
             pids+=($!)
 
-            yarn nx affected -t test lint --base=$NX_BASE --head=$NX_HEAD --parallel=1 &
+            yarn nx affected --target=lint --base=$NX_BASE --head=$NX_HEAD --parallel=3 &
+            pids+=($!)
+            yarn nx affected --target=test --base=$NX_BASE --head=$NX_HEAD --parallel=1 &
             pids+=($!)
-
             (yarn nx affected --target=build --base=$NX_BASE --head=$NX_HEAD --parallel=3 &&
-            npx nx affected --target=e2e --base=$NX_BASE --head=$NX_HEAD --exclude=e2e-storybook,e2e-storybook-angular --parallel=1) &
+            npx nx affected --target=e2e --base=$NX_BASE --head=$NX_HEAD --exclude=e2e-storybook,e2e-storybook-angular,e2e-react-native,e2e-detox --parallel=1) &
```

Comment:
e2e-react-native,e2e-detox no longer have the target e2e, they are changed to e2e-macos now, so they will not got invoked

Reply:
oic, i can remove it then. good to know!